storage/persistent_storage/persistent_storage_local/ways-to-provision-local-storage.adoc

```diff
@@ -0,0 +1,64 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="ways-to-provision-local-storage"]
+= Local storage overview
+include::_attributes/common-attributes.adoc[]
+:context: ways-to-provision-local-storage
+
+toc::[]
+
+You can use either of the following solutions to provision local storage:
```

Comment:
```suggestion
You can use one of the following solutions to provision local storage:
```
I'm not sure how prescriptive we want to be here so this is only a suggestion. I would think that we want to suggest that customers avoid colocating local storage solutions

The alternative would be
```suggestion
You can use any of the following solutions to provision local storage:
```

Let's get a second opinion from @jakobmoellerdev or @DanielFroehlich before committing this change though.

Reply:
I see no reason why we should not allow multiple local storage options. Example use case: use LSO for ODF devices, and LVM for node local storage where the workload handles replication (e.g. Kafka).

Reply:
ack. I think my second example is the correct one in that case. Do we want any sort of note around using these in parallel? `You can use multiple local storage provisioners together but it is not recommended`?

Reply:
Updated it to "You can use any of the following solutions to provision local storage:"